msm/skill_entry.py

```diff
@@ -114,6 +114,13 @@ def __init__(self, name, path, url='', sha='', msm=None):
         self.url = url
         self.sha = sha
         self.msm = msm
+        if msm:
+            u = url.lower()
+            u = u[:-4] if u.endswith(".git") else u
+            self.meta_info = msm.repo.skills_meta_info.get(u, {})
+        else:
+            self.market_info = {}
```

Comment:
Should this be `meta_info` to match the attribute defined above?

Reply:
Good eyes, Indeed it should.